### ISSUE TITLE
[Bug 40020946](https://microsoft.visualstudio.com/OS/_workitems/edit/40020946): [AC UWP Visualizer app sample cards]: Many of the sample cards does not have a 'heading' instead a regular text is present represent the heading for the sample card.

### ISSUE THREAD

**Issue by aaagrawal** [microsoft/AdaptiveCards#7607]
### Problem Description

**Reference:** [Bug 40020946](https://microsoft.visualstudio.com/OS/_workitems/edit/40020946): [AC UWP Visualizer app sample cards]: Many of the sample cards does not have a 'heading' instead a regular text is present represent the heading for the sample card.

### Card JSON

```JSON
**Reference:** [Bug 40020946](https://microsoft.visualstudio.com/OS/_workitems/edit/40020946): [AC UWP Visualizer app sample cards]: Many of the sample cards does not have a 'heading' instead a regular text is present represent the heading for the sample card.
```

**Comment by aaagrawal** [comment]
_Linked to Azure Boards Test Plans_
[### **Design**-Test Case 28646809 Navigate to "StockUpdate.JSON" button](https://microsoft.visualstudio.com/DefaultCollection/OS/_workitems/edit/28646792)
